libraries/rush-lib/src/logic/pnpm/PnpmShrinkwrapFile.ts

```diff
@@ -334,11 +334,7 @@ export class PnpmShrinkwrapFile extends BaseShrinkwrapFile {
     this.overrides = new Map(Object.entries(shrinkwrapJson.overrides || {}));
     this.packageExtensionsChecksum = shrinkwrapJson.packageExtensionsChecksum;
 
-    // Lockfile v9 always has "." in importers filed.
-    this.isWorkspaceCompatible =
-      this.shrinkwrapFileMajorVersion >= ShrinkwrapFileMajorVersion.V9
-        ? this.importers.size > 1
-        : this.importers.size > 0;
```

Comment:
This check was added in https://github.com/microsoft/rushstack/pull/5009. IIRC there was an explicit reason this was necessary. @fzxen?